client/tests/sdk.test.ts

```diff
@@ -9,3 +10,27 @@ describe('getSdk', () => {
     expect(sdk.selectedAddress).toBeTruthy();
   });
 });
+
+describe('return coins to faucet', () => {
```

Comment:
```suggestion
describe('SDK', () => {
```

Since tests are going to be related to the SDK